modules/client-configuration/pages/contact-methods-saltbundle.adoc

```diff
@@ -0,0 +1,81 @@
+[[contact-methods-salt]]
+= Salt Bundle
+
+== What is Salt Bundle?
+
+The Salt Bundle is a single binary package containing Salt Minion, Python 3, required Python modules and libraries.
+
+As the Salt Bundle is shipped with Python 3 and all the requirements for Salt to run, the Salt Bundle is not using the Python version coming from the system, so it can be installed on clients which doesn't meet the requirements for a given Salt version.
+
+It is also possible to install Salt Bundle on the system where Salt Minion is being used already to manage that client by the different Salt Master than the {productname} Server.
+
+== Client Registration with Salt Bundle as a Minion
+
+The registration method with the Salt bundle is the recommended registration method as long as it covers your use case.
```

Comment:
IMO that goes without saying.
```suggestion
The registration method with the Salt bundle is the recommended registration method.
```

Reply:
I think it was the suggestion by @meaksh from the initial docs PR for Salt Bundle. Personally, I like the original more.

Reply:
If we both like the shorter one more, let's see what Pablo says :smiley: 
@meaksh how badly do you want to keep `as long as it covers your use case`?

Reply:
I'm fine removing that. It was not actually me who wrote that. :smile: